app/react/Markdown/components/specs/ContactForm.spec.js

```diff
@@ -23,13 +23,11 @@ describe('ContactForm', () => {
       email: 'peter@parker.com',
       message: 'I miss uncle Ben',
     };
-    props.notify = () => {
-      expect(api.post).toHaveBeenCalledWith('contact', formData);
-      done();
-    };
+
     const component = shallow(<ContactForm {...props} />);
     component.instance().setState(formData);
-    const form = component.find('form');
-    form.simulate('submit', { preventDefault: () => {} });
+    component.find('form').simulate('submit', { preventDefault: () => {} });
+    expect(api.post).toHaveBeenCalledWith('contact', { data: formData, headers: {} });
+    done();
```

Comment:
On the previous implementation, this test actually tested, indirectly, that notify was being called.  I agree it may be somewhat weird.

With the new implementation though, there is nothing asynchronous in this function, so a `done()` in line with everything else has no effect.  This can be re-written to either include the async nature of the api call, or just remove the done(); from here and the argument done in line 20.

Reply:
I opted for removing the done